src/java.base/share/classes/jdk/internal/foreign/abi/DowncallLinker.java

```diff
@@ -41,13 +43,7 @@
 import java.util.List;
 import java.util.stream.Stream;
 
-import jdk.internal.foreign.AbstractMemorySegmentImpl;
-import jdk.internal.foreign.MemorySessionImpl;
-
-import static java.lang.invoke.MethodHandles.collectArguments;
-import static java.lang.invoke.MethodHandles.foldArguments;
-import static java.lang.invoke.MethodHandles.identity;
-import static java.lang.invoke.MethodHandles.insertArguments;
+import static java.lang.invoke.MethodHandles.*;
```

Comment:
It looks inconsistent that in some places you remove the wildcard (*) imports, while in others you add them.

By the way, where does this convention come from? I believe most of the codebase follows the import order: java → empty line → javax → empty line → other jdk packages in alphabetical order → empty line → repeat the same groups for static imports.